CMakeLists.txt

```diff
@@ -104,6 +106,34 @@ if(NOT CUKE_DISABLE_GTEST)
 endif()
 
 #
+# Qt
+#
+
+if(NOT CUKE_DISABLE_QT)
+    set(CMAKE_PREFIX_PATH $ENV{QTDIR})
+    find_package(Qt5Core  QUIET)
+    find_package(Qt5Widgets  QUIET)
+    find_package(Qt5Test  QUIET)
+
+    if(${Qt5Core_FOUND} AND ${Qt5Widgets_FOUND} AND ${Qt5Test_FOUND})
+        message(STATUS "Found Qt version: ${Qt5Core_VERSION_STRING}")
+        include_directories(${Qt5Core_INCLUDE_DIRS} ${Qt5Widgets_INCLUDE_DIRS} ${Qt5Test_INCLUDE_DIRS})
```

Comment:
Please don't add these to the global search path. Instead use `target_include_directories(${target} <PRIVATE|PUBLIC> ${dirs})` for the targets that need it.

Or better yet, only use `target_link_libraries(${target} <PRIVATE|PUBLIC> Qt5::${required_component})` because that already sets up the include path correctly.